CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
-0.77.x (TBD)
+0.78.0 (TBD)
 -------------------
+### Breaking Changes
+* The term `ObjectId` has been replaced with `PrimaryKey` in order to align with the other SDKs. This affects the `[ObjectId]` attribute used to decorate a property.
```

Comment:
Normally yes, but most of the new feature was using the ObjectId nomenclature so introducing it with naming we decided to replace didn't make sense.